include/highfive/bits/H5DataSet_misc.hpp

```
@@ -53,6 +53,10 @@ inline DataSpace DataSet::getSpace() const {
 }
 
 inline DataSpace DataSet::getMemSpace() const { return getSpace(); }
+
+inline size_t DataSet::getOffset() const {
+    return H5Dget_offset(_hid);
```

Could you throw an exception in case H5Dget_offset return HADDR_UNDEF ?